continuationsPlugin/src/test/scala/continuations/ContinuationsPluginSuite.scala

```diff
@@ -135,19 +135,27 @@ class ContinuationsPluginSuite extends FunSuite, CompilerFixtures, StateMachineF
       }
   }
 
-  compilerContext.test("debug".ignore) {
+  compilerContextWithContinuationsPlugin.test("debug".only) {
```

Comment:
You have to remove the .only to get reports on everything in the checks.

Reply:
We should be able to remove the debug test entirely, we have a better handle on how to get the tree outputs at need now.